src/smart-components/OpenChannel/components/OpenChannelMessageList/useHandleOnScrollCallback.ts

```diff
@@ -8,31 +8,38 @@ export interface UseHandleOnScrollCallbackProps {
   scrollRef: React.RefObject<HTMLDivElement>;
 }
 
+export function calcScrollBottom(scrollHeight: number, scrollTop: number): number {
+  return scrollHeight - scrollTop
+}
+
 export function useHandleOnScrollCallback({
   setShowScrollDownButton,
   hasMore,
   onScroll,
   scrollRef,
 }: UseHandleOnScrollCallbackProps): (e: React.UIEvent<HTMLElement>) => void {
   return useCallback((e) => {
+    if (!hasMore) {
```

Comment:
Oh you applied the comment that I left before on Hoon's PR 👏🏻

Reply:
Actually, I had to revert it, its a visual indicator to go to bottom of the page ~